### PR TITLE
add r-seasonal, r-x13binary

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@ndmaxar @RyanHackney agree to be maintainers?

**Comment by rluria14** [comment]
@conda-forge/staged-recipes ready for review

File: recipes/r-seasonal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/seasonal_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/seasonal/seasonal_{{ version }}.tar.gz
+  sha256: 29e8d7c7a01981f52cce5478a03b5ee5074add8af35452446dbd7e1221a0cd27
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: True  # [osx]
```
Comment (dbast) [inline recipes/r-seasonal/meta.yaml:19]: Does not contain compiled code, so the skip can be removed (noarch packages are build on linux after feedstock conversion):
```suggestion
  noarch: generic
```

File: recipes/r-x13binary/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    - "\"%R%\" -e \"library('x13binary')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=x13binary
+  license: GPL-2.0-or-later
+  summary: The US Census Bureau provides a seasonal adjustment program now called 'X-13ARIMA-SEATS'
+    building on both earlier programs called X-11 and X-12 as well as the SEATS program
+    by the Bank of Spain. The US Census Bureau offers both source and binary versions
+    -- which this package integrates for use by other R packages.
+  license_family: OTHER
```
Comment (dbast) [inline recipes/r-x13binary/meta.yaml:46]: Derived from the license, so can be removed:
```suggestion
```

File: recipes/r-x13binary/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/x13binary_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/x13binary/x13binary_{{ version }}.tar.gz
+  sha256: b78383a956061405791439f6991670d2a53626add783cb1f4a391bbbbdf2c8c5
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: True  # [osx]
```
Comment (dbast) [inline recipes/r-x13binary/meta.yaml:19]: Does not contain compiled code, so the skip can be removed (noarch packages are build on linux after feedstock conversion):
```suggestion
  noarch: generic
```

File: recipes/r-x13binary/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    by the Bank of Spain. The US Census Bureau offers both source and binary versions
+    -- which this package integrates for use by other R packages.
+  license_family: OTHER
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - rluria14
+    - ndmaxar
+    - RyanHackney
```
Comment (dbast) [inline recipes/r-x13binary/meta.yaml:53]: Please add the conda-forge/r group:
```suggestion
    - RyanHackney
    - conda-forge/r
```

File: recipes/r-seasonal/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    graphical user interface can be used through the 'seasonalview' package. Uses the
+    X-13-binaries from the 'x13binary' package.
+  license_family: GPL3
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - rluria14
+    - ndmaxar
+    - RyanHackney
```
Comment (dbast) [inline recipes/r-seasonal/meta.yaml:57]: Please add the conda-forge/r team:
```suggestion
    - RyanHackney
    - conda-forge/r
```

File: recipes/r-x13binary/build.sh [hunk 6]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+if [[ ${target_platform} =~ linux.* ]] || [[ ${target_platform} == win-32 ]] || [[ ${target_platform} == win-64 ]] || [[ ${target_platform} == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  mv DESCRIPTION DESCRIPTION.old
+  grep -va '^Priority: ' DESCRIPTION.old > DESCRIPTION
```
Comment (dbast) [inline recipes/r-x13binary/build.sh:6]: Please remove the mv and grep line.

File: recipes/r-seasonal/build.sh [hunk 7]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -o errexit -o pipefail
+
+if [[ ${target_platform} =~ linux.* ]] || [[ ${target_platform} == win-32 ]] || [[ ${target_platform} == win-64 ]] || [[ ${target_platform} == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  mv DESCRIPTION DESCRIPTION.old
+  grep -va '^Priority: ' DESCRIPTION.old > DESCRIPTION
```
Comment (dbast) [inline recipes/r-seasonal/build.sh:8]: Please remove the mv and grep lines.

**Comment by dbast** [comment]
@rluria14 You can make your life easier and use https://github.com/bgruening/conda_r_skeleton_helper

**Comment by rluria14** [comment]
@dbast getting a linking error on osx: `Error: Found multiple instances of libgcc_s.1.dylib: ['lib/libgcc_s.1.dylib', 'lib/R/library/x13binary/lib/libgcc_s.1.dylib']`

any suggestions on how to fix?

**Comment by cbrueffer** [comment]
Haven't seen that error before, but noarch packages are only built on Linux in the feedstock, so it doesn't really matter.

**Comment by rluria14** [comment]
> Haven't seen that error before, but noarch packages are only built on Linux in the feedstock, so it doesn't really matter.

Got it, I wasn't aware of this. So as long as linux passes it should be g2g?

**Comment by cbrueffer** [comment]
Please apply the two remaining changes Daniel suggested about adding `conda-forge/r` as maintainer; then this is good to go.

**Comment by rluria14** [comment]
> Please apply the two remaining changes Daniel suggested about adding `conda-forge/r` as maintainer; then this is good to go.

conda-forge/r has been added as maintainer for both recipes

**Comment by cbrueffer** [comment]
Looks good, thanks!

**Pull request merged**